docs/user-guide/FAQ.md

```diff
@@ -55,6 +55,8 @@ toc:
       url: "#why-do-i-get-not-found-when-i-try-to-start-or-delete-my-pipeline"
     - title: 'How do I override freeze windows to start a build?'
       url: "#how-do-i-override-freeze-windows-to-start-a-build"
+    - title: 'How do I stop a build with FROZEN status?'
```

Comment:
To capture that we are preventing a future scheduled build - `How do I prevent a FROZEN build from scheduling `